_data/conferences.yml

```diff
@@ -25,6 +25,16 @@
 #
 ##########################
 
+- name: DO iOS
+  link: https://do-ios.com/
+  start: 2024-11-13
```

Comment:
The website says it starts on the 12th, but I'm guessing you're not counting the day of workshops as part of the conference?

Reply:
Correct. :)

Reply:
You joining this year by any chance?